.gitignore

```diff
@@ -3,3 +3,11 @@ tags
 deb/
 pkgs
 debian/changelog.*
+*.dep
+/gui-agent/qubes-gui
+/gui-common/qubes-gui-runuser
+*.orig
+*.rej
+*.gz
+*.zstd
+*.tgz
```

Comment:
Those aren't built products (or other files that are expected to be present and should be ignored), so don't add them.